Tooling-Landscape/CapabilityMap/ocToolingAPI_work.yaml

```diff
@@ -180,6 +180,85 @@ paths:
         - casedatastore_auth:
             - write:scan
             - ''
+  /snippet-scan/:
+    get:
+      tags:
+        - snippetScanner
+      summary: Provide URL from which to pull sources for scanning.
+      operationId: uploadSources
+      parameters:
+        - name: purl
+          in: query
+          required: true
+          explode: true
+          schema:
+            type: string
+            example: type/namespace/name@version?qualifiers#subpath
+          description: >-
+            Passes location from which to retrieve sources to be scanned
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/scanRequestId'
+        '403':
+          description: Not authorized
+        '405':
+          description: Validation exception, invalid document
+    post:
+      tags:
+        - snippetScanner
+      summary: Push a single file or bundle of files (tarball, zip) for scanning. Receive an ID for claiming results later
+      operationId: uploadSources
+      requestBody: 
+        description: A collection of sources for scanning.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/sourceBundle'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/sourceBundle'
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/scanRequestId'
+        '403':
+          description: Not authorized
+        '405':
+          description: Validation exception, invalid document
+  /snippet-scan-results/:
+    get:
+      tags:
+        - snippetScanner
+      summary: Retrieve results of previous request
+      operationId: getResults
+      parameters:
+        - name: scanId
+          in: query
+          required: true
+          explode: true
+          schema:
+            $ref: '#/components/schemas/scanRequestId'
+          description: >-
+            Provides Id received from a previous snippet-scan request
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/scanRequestId'
```

Comment:
should return scanResult. We should add a status in the result so that it could be still "in progress" or "failed"